specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/LogicAppsExtension.json

```diff
@@ -0,0 +1,697 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-02-02-preview",
+    "title": "ContainerApps API Client",
+    "description": "Logic app is an extension resource on container app and the extension resource enables logic specific operations on the container app through the APIs listed."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/providers/Microsoft.App/logicApps/{logicAppName}": {
```

Comment:
what about list logic app operatoion?

Reply:
We actualy don't really need that, we can have only one logic app extension per container app and we restrict it to have the same name as the container app